container/logrotate/nginx

```diff
@@ -0,0 +1,15 @@
+/usr/src/app/shared/nginx_log/*.log {
+        daily
+        missingok
+        rotate 30
+        compress
+        delaycompress
+        notifempty
+        create 640 nginx nginx
+        sharedscripts
+        postrotate
+                if [ -f /var/run/nginx.pid ]; then
+                        kill -USR1 `cat /var/run/nginx.pid`
+                fi
```

Comment:
Is this killing the nginx process?

Reply:
It should re-open nginx log files. more [here](http://nginx.org/en/docs/control.html)